src/ralph/settings/base.py

```diff
@@ -339,6 +339,12 @@ def str_to_bool(s: str) -> bool:
 }
 
 # =============================================================================
+# DC view
+# =============================================================================
+
+RACK_LISTING_REVERSE = True
```

Comment:
default should be False. And consider changing name of this variable to better show numeration (from top to bottom or from bottom to top).